packages/web-components/src/components/va-link/va-link.css

```diff
@@ -8,6 +8,10 @@
 :host a {
   cursor: pointer;
   text-decoration: underline;
+  font-size: inherit;
+  font-family: inherit;
+  font-weight: inherit;
+  line-height: inherit;
```

Comment:
I saw this and thought to myself "huh, I always set these individually, I wonder if there's a shorthand css property for this similar to background or border etc". Turns out [there is](https://developer.mozilla.org/en-US/docs/Web/CSS/font)! I had no idea.  I _think_ you could do `font: inherit` here and it would cover all of these? Not sure though :shrug:

Reply:
Oh nice idea! Let me try that...

Reply:
Still works! 🙌🏼

<img width="952" alt="Screenshot 2024-07-11 at 4 43 50 PM" src="https://github.com/user-attachments/assets/6a70ff9d-71c3-4ed4-9be0-a7242c2bc83c">

Reply:
My only thought not doing the shorthand is if there happened to be a font style the design system _didn't_ want to apply? I think leaving it inclusive might should be fine though:

<img width="312" alt="Screenshot 2024-07-11 at 4 46 27 PM" src="https://github.com/user-attachments/assets/d21641c0-d720-4576-8065-3985833e539a">

Reply:
:thinking: true. I guess the scenario there is something changes with the typography typeset we're inheriting from uswds and it propagates to va-link, and it's something we don't want.